source/_integrations/suez_water.markdown

```diff
@@ -24,32 +25,6 @@ It also gets the following data:
 - Last year total consumption
 - Current year total consumption
 
-## Configuration
-
-To add the Suez Water sensor to your installation, add your _Tout Sur Mon Eau_ account credentials and your `counter_id` to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: suez_water
-    username: YOUR_SUEZ_USERNAME
-    password: YOUR_SUEZ_PASSWORD
-    counter_id: YOUR_SUEZ_COUNTER_ID
-```
-
-{% configuration %}
-username:
-  description: The Tout Sur Mon Eau account username.
-  required: true
-  type: string
-password:
-  description: The Tout Sur Mon Eau account password.
-  required: true
-  type: string
-counter_id:
-  description: The Tout Sur Mon Eau counter id. 
-  required: true
-  type: integer
-{% endconfiguration %}
+{% include integrations/config_flow.md %}
 
 `counter_id` is the water counter id. It can be found on your _Tout Sur Mon Eau_ [user account](https://www.toutsurmoneau.fr/mon-compte-en-ligne/historique-de-consommation-tr). Look in the source code of the page for something similar to `url: '/mon-compte-en-ligne/statMData' + '/123456789'`. The `counter_id` in this case is `123456789`.
```

Comment:
No longer needed?

Reply:
Oh right, that's right

Reply:
No wait, you still need it for suez

Reply:
Sorry, I confused it with the other one 👍